layouts/section/job-posting.en.html

```diff
@@ -1,145 +1,154 @@
-            <label class="wb-inv" for="work_link_2"
-              >Additional Work Samples</label
-            >
-            <input
-              type="text"
-              class="form-control"
-              id="work_link_2"
-              name="work_link_2"
-            />
-          </div>
-          <!-- Submit-->
-          <button type="submit" class="button" id="submitButton">
-            <span class="spinner" id="buttonSpinner"
-              ><i
-                style="display:inline-block;margin-right:10px;"
-                class="fas fa-spinner fa-spin"
-              ></i></span
-            >Submit
-          </button>
-          <a href="/legal/privacy-notice">Privacy Notice</a>
-        </form>
+        <h2 class="h2-font-weight">Our hiring process</h2>
+        <h1 class="cds-title heading-title"> At CDS, we welcome and invite difference.</h1>
+        <p>The Canadian Digital Service (CDS) is changing how the federal government designs and delivers digital
+          services. We’re here to improve people’s lives by changing how the government builds technology. We do this by
+          putting people’s needs and concerns front and centre. Every experience Canadians have with their government
+          should be safe, fast, easy, transparent, and accessible.</p>
+        <p>We proudly, passionately, and actively strive to make CDS more reflective of the society that we serve. We
+          will only be able to deliver better, more accessible, and more inclusive public services if we harness diverse
+          thoughts, experiences, and skills. We work hard to create an environment where different perspectives and
+          experiences are valued. We are committed to helping diverse talent thrive.</p>
+        <p>CDS welcomes all applicants, including Veterans and people of all races, ethnicities, religions, sexual
+          orientations, gender identities and expressions, national origins, disabilities, ages, body sizes and
+          including those with diverse households and family commitments. We are committed to providing an inclusive and
+          barrier-free work environment, starting with the hiring process. If you need to be accommodated during any
+          phase of the evaluation process, please use the Contact information below to request specialized
+          accommodation. All information received in relation to accommodation will be kept confidential.</p>
+
+        <div class="cta__link black">
+
+          <a href="/join-our-team/">More about our hiring process<img src="/img/icons/arrow-right-white.svg" /></a>
+        </div>
       </div>
     </div>
-  </section>
-  {{ end }}
-  <!--END APPLY FORM EN -->
+  </div>
 </div>
-{{ end }}
+
+
+<div id="apply-now" class="row work-with-us--apply-now">
+  {{ if .Params.leverId }}
+  <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+    <h2>Apply now</h2>
+    <form enctype="multipart/form-data" class="text-left" role="form" id="contactForm">
```

Comment:
The role is redundant, roles are only for elements that are different than what they are being used for. 

This gets picked up by AXE: 

https://dequeuniversity.com/rules/axe/4.3/aria-allowed-role?application=AxeChrome